### PR TITLE
History preferences fix

### PR DESCRIPTION
**Pull request by stephanfowler**
Keeps the AB test participation and the preference in sync.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/onward/history.js [hunk 1]
```diff
@@ -319,13 +319,13 @@ define([
...
-        return getSummary().showInMegaNav;
+        return getSummary().showInMegaNav !== false;
     }
 
-    function showInMegaNavToggle() {
+    function showInMegaNavEnable(bool) {
         var summary = getSummary();
 
-        summary.showInMegaNav = !summary.showInMegaNav;
+        summary.showInMegaNav = !!bool;
```
Comment (robertberry) [inline static/src/javascripts/projects/common/modules/onward/history.js]: BANG BANG :boom:

**Comment by robertberry** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/onward/history.js (hunk 1 above)
Comment (stephanfowler) [inline static/src/javascripts/projects/common/modules/onward/history.js]: On purpose. `!!` converts to a boolean.

Comment (robertberry) [inline static/src/javascripts/projects/common/modules/onward/history.js]: I know. I just find it amusing.

**Pull request merged**